app/stacks/uk-west/applications-service/terragrunt.hcl

```diff
@@ -17,6 +17,7 @@ dependency "common" {
       applications_service_endpoints = "10.1.3.0/24"
       vpn_gateway                    = "10.1.0.128/25"
     }
+    vnet_id                = "mock_vnet_id"
```

Comment:
This may need to be in the format: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mock_resource_group/providers/Microsoft.Network/virtualNetworks/mock_id`